torch/_inductor/compile_fx.py

```diff
@@ -1632,7 +1632,11 @@ def partition_fn(
         def bw_compiler(
             model: GraphModule, example_inputs: List[InputType]
         ) -> Union[CompiledFxGraph, str]:
-            with dynamo_utils.dynamo_timed("compile_fx.<locals>.bw_compiler"):
+            with dynamo_utils.dynamo_timed(
```

Comment:
I think I also tried to get the full backward time for purposes of logging to dynamo_compile, but I put it here instead: https://github.com/pytorch/pytorch/blob/267641f6f1c011dba7fd5e514d766598035e06dd/torch/_functorch/_aot_autograd/runtime_wrappers.py#L2011

I guess we should pick one?

Reply:
Oh if you have this already I don't need to land mine

Reply:
Yeah, I'm just going to not add it here, and instead have you add it to wherever you had entire_backward_compile.